File: C++/Array 1D.cpp
#include <iostream>
using namespace std;

int main(){
	string nama [3]; //String merupakan tipe data dengan variabel data
	nama[0];
	nama[1];
	nama[2];
	
	cout << "Masukkan Nama Ke 1 : ";
	cin >> nama[0]; 
	// Menginputkan nama ke 1
	
	cout << "Masukkan nama ke 2 : ";
	cin >> nama[1];
	// Menginputkan nama ke 2
	cout << "Masukkan nama ke 3 : ";
	cin >> nama [2];
	// Menginputkan nama ke 3
	
	cout << endl;
	
	cout << "Nama Anak Pertama : "<<nama[0]<<endl;
	// output nama anak pertama
	cout << "Nama Anak Kedua : "<<nama[1]<<endl;
	// output nama anak kedua
	cout << "Nama Anaka ketiga : "<<nama[2]<<endl;
	// output nama anak ketiga
}

File: Java/Array1D.java
package belajar.Java;

import java.util.Scanner;

public class Array1D {
    public static void main (String[] args){
        Scanner name = new Scanner(System.in);
        //Objek dan atribut untuk menginputkan data dengan keyboard
        String[] nama = new String[3];

        System.out.println("Masukkan Nama ke 1 : ");
        nama[0] = name.nextLine();
        //penginputan Nama ke 1
        System.out.println("Masukkan Nama ke 2 : ");
        nama[1] = name.nextLine();
        //Penginputan Nama ke 2
        System.out.println("Masukkan Nama ke 3 : ");
        nama[2] = name.nextLine();
        //Penginputan Nama ke 3

        System.out.println("Nama Anak Pertama : " + nama[0]);
        //Hasil output Ke 1
        System.out.println("Nama Anaka Kedua : " + nama[1]);
        //Hasil output ke 2
        System.out.println("Nama Anak Ketiga : " + nama[2]);
        //Hasil output ke 3

    }
    
}
